tket-py/examples/utils/__init__.py

```diff
@@ -18,7 +18,7 @@ def _repr_hugr(
     def _repr_tk2circ(
         circ: Tk2Circuit, include=None, exclude=None, **kwargs
     ) -> dict[str, bytes | str]:
-        h = Hugr.load_json(circ.to_hugr_json())
+        h = Hugr.from_bytes(circ.to_bytes(EnvelopeConfig))
```

Comment:
`mypy` is complaining because `EnvelopeConfig` is the class, rather than a concrete configuration.
Either pass `EnvelopeConfig.BINARY` or
```suggestion
        h = Hugr.from_bytes(circ.to_bytes())
```
since it defaults to `BINARY`.